cmd/main.go

```diff
@@ -40,6 +40,18 @@ import (
 
 	"github.com/caddyserver/caddy/v2"
 	"github.com/caddyserver/caddy/v2/caddyconfig"
+
+	// For running in minimal environments, this can ease
+	// headaches related to establishing TLS connections.
+	// "Package fallback embeds a set of fallback X.509 trusted
+	// roots in the application by automatically invoking
+	// x509.SetFallbackRoots. This allows the application to
+	// work correctly even if the operating system does not
+	// provide a verifier or system roots pool. ... It's
+	// recommended that only binaries, and not libraries,
+	// import this package. This package must be kept up to
+	// date for security and compatibility reasons."
+	_ "golang.org/x/crypto/x509roots/fallback"
```

Comment:
This was my idea... But does it increase the binary size significantly?

Reply:
The difference in the size of the artifacts is .2-.3 MB (200-300 KB).

Reply:
Oh that's not bad at all